doc/modules/cross_validation.rst

```diff
@@ -209,6 +209,28 @@ The following sections list utilities to generate indices
 that can be used to generate dataset splits according to different cross
 validation strategies.
 
+.. _iid_cv
+
+Cross-validation iterators for i.i.d. data
+==========================================
+
+Independent Identically Distributed (i.i.d.) data is simply the one where
```

Comment:
why "simply"? Let's be objective and drop subjective judgement when defining mathematical concepts.